[2;32m2018-06-22 18:05:38,929 - INFO  [main:ServerBootstrap@99] - connect to database:jdbc:mysql://192.168.1.53:3306/b?useUnicode=true&characterEncoding=UTF-8
[m[2;32m2018-06-22 18:05:39,499 - INFO  [main:ServerBootstrap@125] - Configuration Server bootstrap params.......
[m[2;32m2018-06-22 18:05:39,502 - INFO  [main:ServerBootstrap@126] - Server protocol:	socket
[m[2;32m2018-06-22 18:05:39,502 - INFO  [main:ServerBootstrap@127] - Server port:	9912
[m[2;35m2018-06-22 18:05:39,842 - WARN  [main:XMemcachedClient@727] - XMemcachedClient use Binary protocol
[m[2;32m2018-06-22 18:05:39,880 - INFO  [main:SelectorManager@37] - Creating 8 reactors...
[m[2;35m2018-06-22 18:05:39,890 - WARN  [main:AbstractController@372] - The Controller started at localhost/127.0.0.1:0 ...
[m[2;35m2018-06-22 18:05:39,903 - WARN  [Xmemcached-Reactor-0:MemcachedConnector@251] - Add a session: 127.0.0.1:11211
[m[2;32m2018-06-22 18:05:39,906 - INFO  [main:Application@25] - 初始化AO...
[m[2;32m2018-06-22 18:05:39,944 - INFO  [main:ServerBootstrap@150] - Application PID:	9112
[m[2;35m2018-06-22 18:07:31,914 - WARN  [Thread-62:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :47, action:com.klw.fastfun.pay.common.action.CommonAction, method:queryOrderT
[m[2;35m2018-06-22 18:10:47,970 - WARN  [Thread-12:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :36, action:com.klw.fastfun.pay.common.action.CommonAction, method:queryOrderT
[m[7;31m2018-06-22 18:11:35,917 - ERROR [Xmemcached-Reactor-2:MemcachedHandler@138] - XMemcached network layout exception
[mjava.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at com.google.code.yanf4j.nio.impl.NioTCPSession.readFromBuffer(NioTCPSession.java:203)
	at com.google.code.yanf4j.nio.impl.AbstractNioSession.onRead(AbstractNioSession.java:196)
	at com.google.code.yanf4j.nio.impl.AbstractNioSession.onEvent(AbstractNioSession.java:341)
	at com.google.code.yanf4j.nio.impl.SocketChannelController.dispatchReadEvent(SocketChannelController.java:56)
	at com.google.code.yanf4j.nio.impl.NioController.onRead(NioController.java:157)
	at com.google.code.yanf4j.nio.impl.Reactor.dispatchEvent(Reactor.java:295)
	at com.google.code.yanf4j.nio.impl.Reactor.run(Reactor.java:142)
[2;35m2018-06-22 18:11:35,966 - WARN  [Xmemcached-Reactor-2:MemcachedConnector@365] - Remove a session: 127.0.0.1:11211
[m[7;31m2018-06-22 18:11:37,018 - ERROR [JUICE-SERVER-Handler-4-3-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0xe228e37d, /127.0.0.1:54595 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2018-06-22 18:11:37,019 - ERROR [JUICE-SERVER-Handler-4-3-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0x74bd2371, /127.0.0.1:54596 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2018-06-22 18:11:37,019 - ERROR [JUICE-SERVER-Handler-4-3-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0x933f4c18, /127.0.0.1:54597 :> /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[2;32m2018-06-22 18:11:58,822 - INFO  [main:ServerBootstrap@99] - connect to database:jdbc:mysql://192.168.1.53:3306/b?useUnicode=true&characterEncoding=UTF-8
[m[2;32m2018-06-22 18:11:59,213 - INFO  [main:ServerBootstrap@125] - Configuration Server bootstrap params.......
[m[2;32m2018-06-22 18:11:59,216 - INFO  [main:ServerBootstrap@126] - Server protocol:	socket
[m[2;32m2018-06-22 18:11:59,216 - INFO  [main:ServerBootstrap@127] - Server port:	9912
[m[2;35m2018-06-22 18:11:59,468 - WARN  [main:XMemcachedClient@727] - XMemcachedClient use Binary protocol
[m[2;32m2018-06-22 18:11:59,501 - INFO  [main:SelectorManager@37] - Creating 8 reactors...
[m[2;35m2018-06-22 18:11:59,515 - WARN  [main:AbstractController@372] - The Controller started at localhost/127.0.0.1:0 ...
[m[2;35m2018-06-22 18:11:59,526 - WARN  [Xmemcached-Reactor-0:MemcachedConnector@251] - Add a session: 127.0.0.1:11211
[m[2;32m2018-06-22 18:11:59,528 - INFO  [main:Application@25] - 初始化AO...
[m[2;32m2018-06-22 18:11:59,546 - INFO  [main:ServerBootstrap@150] - Application PID:	16100
[m[2;35m2018-06-22 18:12:18,358 - WARN  [Thread-2:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :59, action:com.klw.fastfun.pay.common.action.CommonAction, method:queryOrderT
[m[2;35m2018-06-22 18:13:55,544 - WARN  [Thread-63:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :15, action:com.klw.fastfun.pay.common.action.CommonAction, method:queryOrderT
[m[2;35m2018-06-22 18:14:23,510 - WARN  [Thread-3:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :24, action:com.klw.fastfun.pay.common.action.CommonAction, method:queryOrderT
[m[2;35m2018-06-22 18:14:23,918 - WARN  [Thread-72:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :18, action:com.klw.fastfun.pay.common.action.CommonAction, method:updateOrderInfo
[m[7;31m2018-06-22 18:20:27,980 - ERROR [JUICE-SERVER-Handler-4-3-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0x7c62c987, /127.0.0.1:54941 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2018-06-22 18:20:27,980 - ERROR [JUICE-SERVER-Handler-4-3-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0x260e087e, /127.0.0.1:54940 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2018-06-22 18:20:27,980 - ERROR [JUICE-SERVER-Handler-4-3-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0x692cb2c6, /127.0.0.1:54939 :> /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[2;32m2018-06-22 18:20:40,420 - INFO  [main:ServerBootstrap@99] - connect to database:jdbc:mysql://192.168.1.53:3306/b?useUnicode=true&characterEncoding=UTF-8
[m[2;32m2018-06-22 18:20:41,249 - INFO  [main:ServerBootstrap@125] - Configuration Server bootstrap params.......
[m[2;32m2018-06-22 18:20:41,250 - INFO  [main:ServerBootstrap@126] - Server protocol:	socket
[m[2;32m2018-06-22 18:20:41,250 - INFO  [main:ServerBootstrap@127] - Server port:	9912
[m[2;35m2018-06-22 18:20:41,620 - WARN  [main:XMemcachedClient@727] - XMemcachedClient use Binary protocol
[m[2;32m2018-06-22 18:20:41,662 - INFO  [main:SelectorManager@37] - Creating 8 reactors...
[m[2;35m2018-06-22 18:20:41,680 - WARN  [main:AbstractController@372] - The Controller started at localhost/127.0.0.1:0 ...
[m[2;35m2018-06-22 18:20:41,696 - WARN  [Xmemcached-Reactor-0:MemcachedConnector@251] - Add a session: 127.0.0.1:11211
[m[2;32m2018-06-22 18:20:41,697 - INFO  [main:Application@25] - 初始化AO...
[m[2;32m2018-06-22 18:20:41,716 - INFO  [main:ServerBootstrap@150] - Application PID:	13484
[m[2;35m2018-06-22 18:26:19,715 - WARN  [Thread-52:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :89, action:com.klw.fastfun.pay.common.action.CommonAction, method:queryOrderT
[m[2;35m2018-06-22 18:26:20,041 - WARN  [Thread-22:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :13, action:com.klw.fastfun.pay.common.action.CommonAction, method:updateOrderInfo
[m[7;31m2018-06-22 18:26:41,633 - ERROR [JUICE-SERVER-Handler-4-3-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0x41887e6d, /127.0.0.1:55490 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2018-06-22 18:26:41,639 - ERROR [JUICE-SERVER-Handler-4-3-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0xad417963, /127.0.0.1:55489 :> /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2018-06-22 18:26:41,633 - ERROR [JUICE-SERVER-Handler-4-3-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0x02c7a11c, /127.0.0.1:55488 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
